### ISSUE TITLE
Age restricted videos can't be downloaded

### ISSUE THREAD

**Issue by Coolkatisa** [Tyrrrz/YoutubeDownloader#540]
### Version

v1.13.3

### Platform

Windows 11

### Steps to reproduce

Step 1: find an age restricted video (One that requires login)
Step 2: attempt to put it into the Youtube Downloader
Step 3: Try to download which will cause the error to pop up

### Details

The error message says that it can't be downloaded due to "sign in"

### Checklist

- [X] I have looked through existing issues to make sure that this bug has not been reported before
- [X] I have provided a descriptive title for this issue
- [X] I have made sure that this bug is reproducible on the latest version of the application
- [X] I have provided all the information needed to reproduce this bug as efficiently as possible
- [ ] I have sponsored this project
- [ ] I have not read any of the above and just checked all the boxes to submit the issue

**Comment by Tyrrrz** [comment]
There are many types of age-restricted videos and some are definitely downloaded. Which one are you having issues with?

**Comment by Coolkatisa** [comment]
The one I am having issues with is this specific video.
https://youtu.be/n2tqE-b6lH4?si=MC5ifXAnePRqtGal

**Comment by DrewzleX** [comment]
Also having the same issue with music marked as age-restricted. The first error says to sign in.  When I sign in I get the following error (attached file).  Thanks!
![Screenshot 2024-11-06 074406](https://github.com/user-attachments/assets/096f8887-332e-459d-b676-8bce599bb9b3)

https://www.youtube.com/watch?v=WZnNTLbogZM&list=PLxA687tYuMWi6_su77B_HWQCTK9fmMSM2&index=10

**Comment by DrewzleX** [comment]
Tried the newest CI build and got the same results.

**Comment by Coolkatisa** [comment]
Sorry it's still not working for you.

**Comment by Heminoid** [comment]
yup same tried with multiple age restricted video ..nothing works

**Comment by DrewzleX** [comment]
Just updated to 1.14 and still having the same issue.  If I am logged out, the tool will tell me to authenticate.  Once authenticated, I receive the same error with a similar message (different line numbers).

![image](https://github.com/user-attachments/assets/9b7cb9ac-09ed-4f6f-94ce-f9d750d532c5)

**Comment by nestg** [comment]
Same problem with any version.

**Comment by Tyrrrz** [comment]
This is a larger issue, not specific to age-restricted videos. Every now and then YouTube may request an unauthenticated client to sign in -- as a measure against bots. This is tracked in #584.

Unfortunately, authentication is currently broken via #546 and #550. Closing as duplicate.
